web/src/cypress/support/schedule.ts

```diff
@@ -124,9 +124,10 @@ function setScheduleTarget(
     }
   }`
   const { scheduleID, target } = tgt
+  const { schedule, ...params } = tgt
```

Comment:
```suggestion
  const { schedule, scheduleID, target, ...params } = tgt
```

Reply:
lol oops

Reply:
oh wait. This was intentional, because params needs to include both `scheduleID` and `target`.

I'll add a comment.